ext/AbstractDifferentiationZygoteExt.jl

```diff
@@ -7,6 +7,10 @@ else
     using ..Zygote: Zygote
 end
 
-AD.ZygoteBackend() = AD.ReverseRuleConfigBackend(Zygote.ZygoteRuleConfig())
+AD.@primitive function pullback_function(ba::AD.ZygoteBackend, f, xs...)
+    @inline # @primitive doesn't support this before function keyword
```

Comment:
This syntax is not supported on Julia 1.6.

Reply:
```suggestion
```

Reply:
That's odd, I thought I force-pushed a new commit which removed it. Either way, #77 is the way to go.